Perun (#136)
hyperledger-labs/hyperledger-labs.github.io: labs/perun-framework.md

# Lab Name

Perun Framework

# Short Description

Blockchain-agnostic State Channels framework.

# Scope of Lab

> The Perun Framework is suite of implementations around the [Perun state

Reviewer: ```suggestion
The Perun Framework is a suite of implementations around the [Perun state
```
typo, missing article

Author: fixed in 484ffbe

# Lab Name

Perun Framework

# Short Description

> Blockchain-agnostic State Channels framework, written in Go.

Reviewer: ```suggestion
Blockchain-agnostic State Channels framework.
```
see other suggestion for comment

Author: Changed in commit 3a1a883.

# Scope of Lab

> Perun Framework is a Go implementation of the [Perun state

Reviewer: ```suggestion
The Perun Framework is a suite of implementations around the [Perun state
```
I would not mention implementation details like Go in the general description of the Perun framework. For example, The Ethereum smart contracts are written in Solidity, its tests are written in TypeScript etc. In the future, we might also add additional implementations in other languages.

Author: @sebastianst, thank you for the review! Your suggested change makes sense. I've added it in commit 3a1a883.
